js/interface.js

```diff
@@ -62,36 +62,39 @@ function parseThumbnail(file) {
 
 function navigateToDefaultFolder() {
   if (typeof data === 'undefined' || !data || !data.appId) {
-    // No folder was specified
+// No folder was specified
     return;
   }
-
   var $listHolder;
   var folderId;
   var type;
   var $el = $('[data-app-id="' + data.appId + '"][data-browse-folder]');
 
-  // Activate folder on left sidebar
+// Activate folder on left sidebar
   if ($el.data('type') === 'organization') {
     $listHolder = $el;
   } else {
     $listHolder = $el.find('.list-holder');
   }
-
+  
   $('.dropdown-menu-holder').find('.list-holder.active').removeClass('active');
   $listHolder.first().addClass('active');
 
-  // Set first folder of breadcrumbs
+// Set first folder of breadcrumbs
   resetUpTo($el);
-
+  
+  if(data.appId) {
```

Comment:
Did you check the code guidelines? There should be a space after the `if` keyword.

Reply:
done